apps/web/src/components/tx/ConfirmTxDetails/TxDetails.tsx

```diff
@@ -157,6 +155,43 @@ export const TxDetails = ({ safeTx, txData, showHashes }: TxDetailsProps) => {
                 </TxDetailsRow>
 
                 <TxDetailsRow label="Nonce">{safeTx.data.nonce}</TxDetailsRow>
+
+                <Button onClick={() => setShowHashes(!showHashes)} sx={{ all: 'unset' }}>
+                  <Typography
+                    variant="body2"
+                    fontWeight={700}
+                    display="inline-flex"
+                    alignItems="center"
+                    color="primary.light"
+                    sx={{ cursor: 'pointer' }}
+                  >
+                    Transaction hashes <ExpandMoreIcon sx={{ transform: !showHashes ? 'rotate(180deg)' : undefined }} />
+                  </Typography>
+                </Button>
+
+                {showHashes && domainHash && (
+                  <TxDetailsRow label="Domain hash">
+                    <Typography variant="body2" width="100%" sx={{ wordWrap: 'break-word' }}>
+                      <HexEncodedData hexData={domainHash} limit={66} highlightFirstBytes={false} />
+                    </Typography>
+                  </TxDetailsRow>
+                )}
+
+                {showHashes && messageHash && (
+                  <TxDetailsRow label="Message hash">
+                    <Typography variant="body2" width="100%" sx={{ wordWrap: 'break-word' }}>
+                      <HexEncodedData hexData={messageHash} limit={66} highlightFirstBytes={false} />
+                    </Typography>
+                  </TxDetailsRow>
+                )}
+
+                {showHashes && safeTxHash && (
+                  <TxDetailsRow label="safeTxHash">
+                    <Typography variant="body2" width="100%" sx={{ wordWrap: 'break-word' }}>
+                      <HexEncodedData hexData={safeTxHash} limit={66} highlightFirstBytes={false} />
```

Comment:
highlightFirstBytes is false by default.

Reply:
It [defaults to `true`](https://github.com/safe-global/safe-wallet-monorepo/blob/dev/apps/web/src/components/transactions/HexEncodedData/index.tsx#L18).